scripts/build_airocean_parameters.py

```diff
@@ -0,0 +1,558 @@
+#!/usr/bin/env python
+###############################################################################
+# $Id$
+#
+#  Project:  PROJ
+#  Purpose:  Build airocean parameters
+#  Author:   Pierre Louvart <pierre.louvart at ifremer.fr>
+#
+###############################################################################
+#  Copyright (c) 2025, Pierre Louvart <pierre.louvart at ifremer.fr>
+#
+#  Permission is hereby granted, free of charge, to any person obtaining a
+#  copy of this software and associated documentation files (the "Software"),
+#  to deal in the Software without restriction, including without limitation
+#  the rights to use, copy, modify, merge, publish, distribute, sublicense,
+#  and/or sell copies of the Software, and to permit persons to whom the
+#  Software is furnished to do so, subject to the following conditions:
+#
+#  The above copyright notice and this permission notice shall be included
+#  in all copies or substantial portions of the Software.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+#  OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+#  THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#  DEALINGS IN THE SOFTWARE.
+###############################################################################
+
+"""
+    This python script can be launched "as is" with no additional argument.
+    It will generate 2 plots (require geopandas and matplotlib):
+    - one for the 20 faces unfolded net
+    - one for the complete 23 faces unfolded net that makes up the final airocean map
+    It will also print on the standard output the C code initializing vertices, faces and
+    transform matrices (forward and inverse) required to conduct the airocean projection.
+"""
+
+
+import numpy as np
+from pathlib import Path
+import json
+
+
+"""
+    Definition of the vertices of the unfolded icosahedron net:
+"""
+
+
+# Icosahedron properties
+circumscribed_radius = 1
+inscribed_radius = 3**0.5 / 12 * (3 + 5**0.5)
+triangle_edge = circumscribed_radius / np.sin(2 * np.pi / 5)
```

Comment:
keep a native Python float (rather than mixing np.float64) by a cast:
```suggestion
triangle_edge = float(circumscribed_radius / np.sin(2 * np.pi / 5))
```